Tests/test_content.py

```diff
@@ -293,6 +366,13 @@ def main():
         print('no integrations are configured for test')
         return
 
+    with open('public_ip', 'rb') as f:
+        public_ip = f.read()
+
+    ami = AMIConnection(public_ip)
+    ami.clone_mock_data()  # FUTURE: pull instead of clone
```

Comment:
check with devops if possible to clone on AMI creation, otherwise remove comment.